swan-lake/development-tutorials/build-and-run/cli-commands.md

```diff
@@ -61,21 +62,26 @@ COMMANDS
         format          Format Ballerina source files
         grpc            Generate the Ballerina sources for a given Protocol
                         Buffer definition
-        graphql         Generate the Ballerina client sources for a GraphQL config file,
-                        generate the GraphQL schema for a Ballerina GraphQL service, and 
-                        generate the Ballerina GraphQL service for a GraphQL schema
+        graphql         Generate the Ballerina client sources for a GraphQL config file, the GraphQL schema for a GraphQL service, and Ballerina service sources for a GraphQL schema
```

Comment:
It seems this line is longer than others. Shall we break this line? (like in OpenApi)

Reply:
IIRC there is a character limit per line. @praneesha can check this?

Reply:
Updated.